test/runtests.jl

```diff
@@ -7,7 +7,7 @@ if isinteractive()
     @info ("In interactive use, one should respond \"n\" when the program"
            * " offers to create or replace files associated with some tests.")
 else
-    @info ("Four tests should correctly report failure in the transcript"
+    @info ("Eight tests should correctly report failure in the transcript"
```

Comment:
how did you count to get to "four" ? if I count all the "complaints" about failing test or non-existing files i got to "seven" before the additional one in this PR. (thought its quite hard to count because the visual separation isn't too great)